dotCMS/src/integration-test/java/com/dotcms/datagen/TestDataUtils.java

```diff
@@ -1773,4 +1793,183 @@ public static Contentlet createNewsLikeURLMappedContent(final String newsPattern
                         newsContentType.id(), host, sysPublishDate);
     }
 
+    public static ContentType getMultipleBinariesContentType() {
+        return getMultipleBinariesContentType("MultipleBinaries" + System.currentTimeMillis(), APILocator.systemHost(),null);
+    }
+
+    /**
+     * This will give you a CT that has 3 non-required binaries.
+     * The default metadata generated gets removed.
+     * @param contentTypeName
+     * @param site
+     * @param workflowIds
+     * @return
+     */
+    @WrapInTransaction
+    public static ContentType getMultipleBinariesContentType(final String contentTypeName,
+            final Host site,
+            final Set<String> workflowIds) {
+
+        ContentType contentType = null;
+        try {
+            try {
+                contentType = APILocator.getContentTypeAPI(APILocator.systemUser())
+                        .find(contentTypeName);
+            } catch (NotFoundInDbException e) {
+                //Do nothing...
+            }
+            if (contentType == null) {
+
+                final List<com.dotcms.contenttype.model.field.Field> fields = new ArrayList<>();
+                if (null != site) {
+                    fields.add(
+                            new FieldDataGen()
+                                    .name("Site or Folder")
+                                    .velocityVarName("hostfolder")
+                                    .sortOrder(1)
+                                    .required(Boolean.TRUE)
+                                    .type(HostFolderField.class)
+                                    .next()
+                    );
+                }
+
+                fields.add(
+                        new FieldDataGen()
+                                .name("Title")
+                                .velocityVarName("title")
+                                .sortOrder(2)
+                                .type(TextField.class)
+                                .next()
+                );
+
+                fields.add(
+                        new FieldDataGen()
+                                .name("fileAsset1")
+                                .velocityVarName("fileAsset1")
+                                .sortOrder(3)
+                                //The only way we can guarantee a field won't be indexed is by setting all these to false
+                                .indexed(false).searchable(false).listed(false).unique(false)
+                                .type(BinaryField.class)
+                                .next()
+                );
+
+                fields.add(
+                        new FieldDataGen()
+                                .name("fileAsset2")
```

Comment:
Codacy found an issue: [The String literal "fileAsset2" appears 4 times in this file; the first occurrence is on line 1,858](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6220272)